src/Control/Inciter/InputDeck/Grammar.hpp

```diff
@@ -733,7 +757,9 @@ namespace deck {
                            bc< kw::bc_dirichlet, tag::compflow, tag::bcdir >,
                            bc< kw::bc_sym, tag::compflow, tag::bcsym >,
                            bc< kw::bc_inlet, tag::compflow, tag::bcinlet >,
-                           bc< kw::bc_outlet, tag::compflow, tag::bcsubsonicoutlet >,
+                           subsonic_bc< kw::bc_outlet,
+                                        tag::compflow,
+                                        tag::bcoutlet >,
```

Comment:
This is what puts in the grammar/parsing block, specific to outlet + farfield pressure, _only_ for compflow and not for transport or multimat (at this point).